### PR TITLE
Make placement_group=None work as expected.

### PR DESCRIPTION
**Pull request by ericl**
Fixes https://github.com/ray-project/ray/pull/16049

### CODE REVIEW

File: python/ray/actor.py [hunk 1]
```diff
@@ -431,7 +431,7 @@ def options(self,
                 max_task_retries=None,
                 name=None,
                 lifetime=None,
-                placement_group=None,
+                placement_group="default",
```
Comment (richardliaw) [inline python/ray/actor.py:434]: can you add docstring?

**Comment by rkooo567** [comment]
Can you add a test change in this file? https://github.com/ray-project/ray/pull/16049/files (this was the reason why this issue wasn't caught)

**Comment by ericl** [comment]
Done

**Comment by richardliaw** [comment]
LGTM but @rkooo567 should approve

**Review by rkooo567 — APPROVED** [review]

**Comment by rkooo567** [comment]
should we pick this up for 1.4.1?

**Pull request merged**